.github/CODEOWNERS

```diff
@@ -2,12 +2,12 @@
 
 # Ensure that the team responsible for upgrades sees any PRs that would
 # add GitHub-hosted dependencies to that platform.
-requirements/edx/github.in                                          @openedx/arbi-bom
+requirements/edx/github.in                                          @openedx/2u-arbi-bom
 
 # Core
 common/djangoapps/student/
-common/djangoapps/student/models/__init__.py                        @openedx/teaching-and-learning
-common/djangoapps/student/models/course_enrollment.py               @openedx/teaching-and-learning
+common/djangoapps/student/models/__init__.py                        @openedx/2u-tnl
+common/djangoapps/student/models/course_enrollment.py               @openedx/2u-tnl
```

Comment:
@feanil , the error here states that 2u-tnl needs write access. Right now they [have read access](https://github.com/openedx/edx-platform/settings/access) on the repo. When I bump the access level up to "write", the error here goes away.

Reply:
Ah, Kyle, I misunderstood your question, I thought it was about adding people as reviewers and not for adding them as CODEOWNERS.  Yes, you're right they do need `write` access and I think it's fine to add them explicitly to the list of collaborators for edx-platform.